api/v1/framework/cypress.schema.json

```diff
@@ -0,0 +1,210 @@
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "Name of the reporter. You may have to install the associated npm dependency through npm.packages field.",
+                "type": "string",
+                "examples": [
+                  "dot",
+                  "nyan"
+                ]
+              },
+              "options": {
+                "description": "Options to pass to the reporter."
+              }
+            },
+            "additionalProperties": false
+          }
+        }
+      },
+      "required": [
+        "configFile",
+        "version"
+      ],
+      "additionalProperties": false
+    },
+    "env": {
+      "$ref": "../subschema/common.schema.json#/definitions/env"
+    },
+    "suites": {
+      "description": "The set of properties providing details about the test suites to run.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The name of the test suite, which will be reflected in the test results in Sauce Labs.",
+            "type": "string"
+          },
+          "browser": {
+            "$ref": "../subschema/common.schema.json#/definitions/browser",
+            "enum": [
+              "chrome",
+              "firefox",
+              "microsoftedge"
+            ]
+          },
+          "browserVersion": {
+            "$ref": "../subschema/common.schema.json#/definitions/browserVersion"
+          },
+          "platformName": {
+            "$ref": "../subschema/common.schema.json#/definitions/platformName",
+            "enum": [
+              "macOS 11.00",
+              "macOS 12",
+              "Windows 10",
+              "Windows 11"
+            ]
+          },
+          "screenResolution": {
+            "$ref": "../subschema/common.schema.json#/definitions/screenResolution"
+          },
+          "config": {
+            "description": "Provides details related to the Cypress test configuration.",
+            "type": "object",
+            "properties": {
+              "testingType": {
+                "description": "Specify the type of tests to execute; either e2e or component. Defaults to e2e",
+                "enum": [
+                  "e2e",
+                  "component"
+                ],
+                "default": "e2e"
+              },
+              "specPattern": {
+                "description": "One or more paths to the Cypress test files to run for this suite.",
+                "type": "array"
+              },
+              "excludeSpecPattern": {
+                "description": "Exclude test files for this suite.",
+                "type": "array"
+              }, 
+              "env": {
+                "$ref": "../subschema/common.schema.json#/definitions/env"
+              }
+            },
+            "required": [
+              "testFiles"
```

Comment:
That's not correct anymore. We require at least one of `specPattern` or `excludeSpecPattern` now.